Detectors/ITSMFT/common/reconstruction/src/DigitPixelReader.cxx

```diff
@@ -98,24 +98,20 @@ void DigitPixelReader::openInput(const std::string inpName, o2::detectors::DetID
     LOG(FATAL) << "Failed to find " << (detName + "Digit").c_str() << " branch in the " << mInputTree->GetName()
                << " from file " << inpName;
   }
-  setDigits(mDigitsSelf);
 
   mInputTree->SetBranchAddress((detName + "DigitROF").c_str(), &mROFRecVecSelf);
   if (!mROFRecVecSelf) {
     LOG(FATAL) << "Failed to find " << (detName + "DigitROF").c_str() << " branch in the " << mInputTree->GetName()
                << " from file " << inpName;
   }
-  setROFRecords(mROFRecVecSelf);
 
   mInputTree->SetBranchAddress((detName + "DigitMC2ROF").c_str(), &mMC2ROFRecVecSelf);
   if (!mMC2ROFRecVecSelf) {
     LOG(FATAL) << "Failed to find " << (detName + "DigitMC2ROF").c_str() << " branch in the " << mInputTree->GetName()
                << " from file " << inpName;
   }
-  setMC2ROFRecords(mMC2ROFRecVecSelf);
 
   mInputTree->SetBranchAddress((detName + "DigitMCTruth").data(), &mDigitsMCTruthSelf);
-  setDigitsMCTruth(mDigitsMCTruthSelf);
```

Comment:
@shahor02 This line should be restored.  Otherwise, the labels do not propagated to the file with clusters.